core/crypto/sm3-hash.c

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2018, ARM Limited
+ * Copyright (C) 2019, Linaro Limited
+ * Copyright (C) 2019 Huawei Technologies Co., Ltd
```

Comment:
minor: a coma after 2019?

Reply:
Since a comma is not consistently used, I'd rather keep the Huawei notices consistent at least, and never use a comma for them.

Reply:
ok